drv/cosmo-hf/src/main.rs

```diff
@@ -50,14 +50,21 @@ fn main() -> ! {
     let seq =
         drv_spartan7_loader_api::Spartan7Loader::from(LOADER.get_task_id());
 
-    let drv = FlashDriver {
+    let mut drv = FlashDriver {
         drv: fmc_periph::SpiNor::new(seq.get_token()),
     };
     drv.flash_set_quad_enable();
 
-    // Check the flash chip's ID against Table 7.3.1 in the datasheet
+    // Check the flash chip's ID against Table 7.3.1 in the W25Q01JV datasheet.
```

Comment:
I would include which revision of the datasheet this is. Especially as my copy of Rev E (March 04, 2024) has this as section 6.3.1 even thought the ToC is 7.3.1. I would also note that these IDs are specific to the JEDEC ID command as the table points out other values that are different from other commands.

Reply:
Done in 21caaea